### PR TITLE
login user supervisor

### PR DESCRIPTION
**Pull request by n-luc**
Screenshot from 2019-04-22 17-14-08
Screenshot from 2019-04-22 17-20-54

### CODE REVIEW

**Comment by n-luc** [comment]
https://edu-redmine.sun-asterisk.vn/issues/10850

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -14,22 +14,26 @@
...
-      <a class="btn btn-outline-primary" href="/login" role="button">
-      Login
-      </a>
-      <div style="padding-left: 10px;">
-        <a class="btn btn-outline-success" href="/signup" role="button">
-        Sign Up
-        </a>
-      </div>
+      <% if user_signed_in? %>
+        <%= link_to "Log Out", destroy_user_session_path, method: :delete, class: "btn btn-outline-primary" %>
```
Comment (dothidiemthao) [inline app/views/layouts/_header.html.erb]: i18n nhé
fix cho các line trong file này

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <th><%= t ".id" %></th>
+        <th><%= t ".name" %></th>
+        <th><%= t ".email" %></th>
+        <th><%= t ".action" %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= render @users %>
+    </tbody>
+    <%= paginate @users, theme: 'twitter-bootstrap-4' %>
```
Comment (dothidiemthao) [inline app/views/users/index.html.erb]: dùng nháy "

File: config/environments/development.rb [hunk 3]
```diff
@@ -27,6 +27,8 @@
     config.cache_store = :null_store
   end
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (dothidiemthao) [inline config/environments/development.rb]: nháy "

**Review by dothidiemthao — APPROVED** [review]

**Pull request merged**